frontend/src/scenes/billing/CurrentUsage.tsx

```diff
@@ -61,7 +61,7 @@ export function CurrentUsage(): JSX.Element | null {
                             'Your current plan has an unlimited event allocation.'}
                         <Progress
                             type="line"
-                            percent={percentage !== null ? percentage * 100 : 100}
+                            percent={percentage !== null ? Math.floor(percentage * 100) : 100}
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed ?

Reply:
We want floor here rather than rounding - 99.99% is not yet 100% :)